packages/ui-kit/src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { VFC, ReactNode } from "react";
+import clsx from "clsx";
+
+import styles from "./IconButton.module.css";
+import { Button, ButtonProps, ButtonLabel } from "../Button/Button";
+
+export interface IconButtonProps extends Omit<ButtonProps, "variant"> {
+  icon: ReactNode;
+  reverse?: boolean;
```

Comment:
I'd rather call it `alignment` with `left` and `right` props, since there is not saying if left is actually the default. Since we also could use it in other places we could have a shared type like `HorizontalAlignment`